bin/alluxio

```diff
@@ -21,6 +21,7 @@ function printUsage {
   echo -e "  bootstrapConf \t Generate a config file if one doesn't exist"
   echo -e "  fs            \t Command line tool for interacting with the Alluxio filesystem."
   echo -e "  getConf [key] \t Look up a configuration key, or print all configuration."
+  echo -e "  getConfKey [name] \t Print a configuration key given an environment variable name."
```

Comment:
this could mislead users to think that environment variables can be used to configure alluxio servers. Can we hide the API by calling `bin/alluxio runClass alluxio.cli.GetConfKey` directly?

Reply:
Makes sense. Will do.